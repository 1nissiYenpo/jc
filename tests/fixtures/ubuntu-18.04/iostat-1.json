[{"percent_user":11.66,"percent_nice":3.51,"percent_system":7.44,"percent_iowait":1.33,"percent_steal":0.0,"percent_idle":76.06,"type":"cpu"},{"device":"loop0","tps":0.05,"kb_read_s":0.16,"kb_wrtn_s":0.0,"kb_read":125,"kb_wrtn":0,"type":"device"},{"device":"loop1","tps":11.06,"kb_read_s":12.71,"kb_wrtn_s":0.0,"kb_read":10139,"kb_wrtn":0,"type":"device"},{"device":"loop2","tps":0.12,"kb_read_s":1.77,"kb_wrtn_s":0.0,"kb_read":1413,"kb_wrtn":0,"type":"device"},{"device":"loop3","tps":0.07,"kb_read_s":0.45,"kb_wrtn_s":0.0,"kb_read":359,"kb_wrtn":0,"type":"device"},{"device":"loop4","tps":0.07,"kb_read_s":1.34,"kb_wrtn_s":0.0,"kb_read":1067,"kb_wrtn":0,"type":"device"},{"device":"loop5","tps":0.13,"kb_read_s":1.49,"kb_wrtn_s":0.0,"kb_read":1190,"kb_wrtn":0,"type":"device"},{"device":"loop6","tps":0.05,"kb_read_s":0.16,"kb_wrtn_s":0.0,"kb_read":125,"kb_wrtn":0,"type":"device"},{"device":"loop7","tps":0.08,"kb_read_s":1.34,"kb_wrtn_s":0.0,"kb_read":1070,"kb_wrtn":0,"type":"device"},{"device":"fd0","tps":0.26,"kb_read_s":1.01,"kb_wrtn_s":0.0,"kb_read":806,"kb_wrtn":0,"type":"device"},{"device":"sda","tps":82.01,"kb_read_s":1901.45,"kb_wrtn_s":3368.25,"kb_read":1517318,"kb_wrtn":2687800,"type":"device"},{"device":"scd0","tps":2.89,"kb_read_s":64.94,"kb_wrtn_s":0.0,"kb_read":51818,"kb_wrtn":0,"type":"device"},{"device":"scd1","tps":2.89,"kb_read_s":64.44,"kb_wrtn_s":0.0,"kb_read":51418,"kb_wrtn":0,"type":"device"},{"device":"loop8","tps":15.88,"kb_read_s":17.15,"kb_wrtn_s":0.0,"kb_read":13683,"kb_wrtn":0,"type":"device"},{"device":"loop9","tps":0.08,"kb_read_s":0.45,"kb_wrtn_s":0.0,"kb_read":362,"kb_wrtn":0,"type":"device"},{"device":"loop10","tps":0.12,"kb_read_s":0.6,"kb_wrtn_s":0.0,"kb_read":476,"kb_wrtn":0,"type":"device"},{"device":"loop11","tps":0.04,"kb_read_s":0.14,"kb_wrtn_s":0.0,"kb_read":108,"kb_wrtn":0,"type":"device"},{"device":"loop12","tps":0.06,"kb_read_s":0.44,"kb_wrtn_s":0.0,"kb_read":353,"kb_wrtn":0,"type":"device"},{"device":"loop13","tps":0.05,"kb_read_s":0.44,"kb_wrtn_s":0.0,"kb_read":348,"kb_wrtn":0,"type":"device"},{"device":"loop14","tps":0.08,"kb_read_s":0.46,"kb_wrtn_s":0.0,"kb_read":366,"kb_wrtn":0,"type":"device"},{"device":"loop15","tps":0.08,"kb_read_s":0.45,"kb_wrtn_s":0.0,"kb_read":361,"kb_wrtn":0,"type":"device"},{"percent_user":1.01,"percent_nice":0.0,"percent_system":0.0,"percent_iowait":0.0,"percent_steal":0.0,"percent_idle":98.99,"type":"cpu"},{"device":"loop0","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop1","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop2","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop3","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop4","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop5","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop6","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop7","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"fd0","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"sda","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"scd0","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"scd1","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop8","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop9","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop10","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop11","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop12","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop13","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop14","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop15","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"percent_user":0.0,"percent_nice":0.0,"percent_system":1.0,"percent_iowait":0.0,"percent_steal":0.0,"percent_idle":99.0,"type":"cpu"},{"device":"loop0","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop1","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop2","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop3","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop4","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop5","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop6","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop7","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"fd0","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"sda","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"scd0","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"scd1","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop8","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop9","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop10","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop11","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop12","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop13","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop14","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"},{"device":"loop15","tps":0.0,"kb_read_s":0.0,"kb_wrtn_s":0.0,"kb_read":0,"kb_wrtn":0,"type":"device"}]
