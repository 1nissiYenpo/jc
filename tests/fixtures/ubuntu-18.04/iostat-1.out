Linux 4.15.0-158-generic (kbrazil-ubuntu) 	12/01/2021 	_x86_64_	(1 CPU)

avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.66    3.51    7.44    1.33    0.00   76.06

Device             tps    kB_read/s    kB_wrtn/s    kB_read    kB_wrtn
loop0             0.05         0.16         0.00        125          0
loop1            11.06        12.71         0.00      10139          0
loop2             0.12         1.77         0.00       1413          0
loop3             0.07         0.45         0.00        359          0
loop4             0.07         1.34         0.00       1067          0
loop5             0.13         1.49         0.00       1190          0
loop6             0.05         0.16         0.00        125          0
loop7             0.08         1.34         0.00       1070          0
fd0               0.26         1.01         0.00        806          0
sda              82.01      1901.45      3368.25    1517318    2687800
scd0              2.89        64.94         0.00      51818          0
scd1              2.89        64.44         0.00      51418          0
loop8            15.88        17.15         0.00      13683          0
loop9             0.08         0.45         0.00        362          0
loop10            0.12         0.60         0.00        476          0
loop11            0.04         0.14         0.00        108          0
loop12            0.06         0.44         0.00        353          0
loop13            0.05         0.44         0.00        348          0
loop14            0.08         0.46         0.00        366          0
loop15            0.08         0.45         0.00        361          0

avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           1.01    0.00    0.00    0.00    0.00   98.99

Device             tps    kB_read/s    kB_wrtn/s    kB_read    kB_wrtn
loop0             0.00         0.00         0.00          0          0
loop1             0.00         0.00         0.00          0          0
loop2             0.00         0.00         0.00          0          0
loop3             0.00         0.00         0.00          0          0
loop4             0.00         0.00         0.00          0          0
loop5             0.00         0.00         0.00          0          0
loop6             0.00         0.00         0.00          0          0
loop7             0.00         0.00         0.00          0          0
fd0               0.00         0.00         0.00          0          0
sda               0.00         0.00         0.00          0          0
scd0              0.00         0.00         0.00          0          0
scd1              0.00         0.00         0.00          0          0
loop8             0.00         0.00         0.00          0          0
loop9             0.00         0.00         0.00          0          0
loop10            0.00         0.00         0.00          0          0
loop11            0.00         0.00         0.00          0          0
loop12            0.00         0.00         0.00          0          0
loop13            0.00         0.00         0.00          0          0
loop14            0.00         0.00         0.00          0          0
loop15            0.00         0.00         0.00          0          0

avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    1.00    0.00    0.00   99.00

Device             tps    kB_read/s    kB_wrtn/s    kB_read    kB_wrtn
loop0             0.00         0.00         0.00          0          0
loop1             0.00         0.00         0.00          0          0
loop2             0.00         0.00         0.00          0          0
loop3             0.00         0.00         0.00          0          0
loop4             0.00         0.00         0.00          0          0
loop5             0.00         0.00         0.00          0          0
loop6             0.00         0.00         0.00          0          0
loop7             0.00         0.00         0.00          0          0
fd0               0.00         0.00         0.00          0          0
sda               0.00         0.00         0.00          0          0
scd0              0.00         0.00         0.00          0          0
scd1              0.00         0.00         0.00          0          0
loop8             0.00         0.00         0.00          0          0
loop9             0.00         0.00         0.00          0          0
loop10            0.00         0.00         0.00          0          0
loop11            0.00         0.00         0.00          0          0
loop12            0.00         0.00         0.00          0          0
loop13            0.00         0.00         0.00          0          0
loop14            0.00         0.00         0.00          0          0
loop15            0.00         0.00         0.00          0          0

